gcc/rust/util/rust-unicode.h

```diff
@@ -19,10 +19,29 @@
 #ifndef RUST_UNICODE_H
 #define RUST_UNICODE_H
 
+#include "optional.h"
 #include "rust-system.h"
+#include "rust-lex.h"
 
 namespace Rust {
 
+class Utf8String
+{
+private:
```

Comment:
I was about to say a nit here to move the private fields to the bottom of the class declaration but then i realised we do it this style in AST and HIR classes.

What is the GCC style? I think having them at the top like this is probably the best now when i think about it.

@dkm @CohenArthur @tschwinge any opinions on the style of putting private fields at the top of the class or bottom?

Reply:
I like having private fields at the bottom of my classes personally. I never gave it much attention haha